pkg/virt-api/webhooks/validating-webhook/admitters/vmclone-admitter.go

```diff
@@ -25,6 +25,8 @@ import (
 	"fmt"
 	"strings"
 
+	"kubevirt.io/kubevirt/pkg/network/link"
```

Comment:
nit: Consider moving this import to the last group that imports `kubevirt.io/kubevirt/pkg/*`.

Reply:
sorry I can't grup you

Reply:
```go
...
	"kubevirt.io/kubevirt/pkg/network/link"
	webhookutils "kubevirt.io/kubevirt/pkg/util/webhooks"
	virtconfig "kubevirt.io/kubevirt/pkg/virt-config"
```

Reply:
I see, I use the goland, the goland add it automatically.

Reply:
Yes, I have tried to find a formatter that takes care of this and could not find one.